CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * [#293](https://github.com/slack-ruby/slack-ruby-client/pull/293): Rubocop auto-correct and comprehensive todo - [@jcraigk](https://github.com/jcraigk).
 * [#297](https://github.com/slack-ruby/slack-ruby-client/pull/297): Various Rubocop fixes - [@jcraigk](https://github.com/jcraigk).
 * [#298](https://github.com/slack-ruby/slack-ruby-client/pull/298): Add `admin.apps`, `admin.app.requests` and `views` endpoints - [@jmanian](https://github.com/jmanian).
+* [#299](https://github.com/slack-ruby/slack-ruby-client/issues/299): files_list does not paginate - [@rangerscience](https://github.com/rangerscience)
```

Comment:
Danger is complaining that the text doesn't start with a capital letter, so make it "Added pagination support to `file_list`."